Gemfile

```diff
@@ -22,7 +22,9 @@ group :test do
 end
 
 group :integration do
-  gem 'berkshelf', '~> 5.2'
+  if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.4')
+    gem 'berkshelf', '~> 7'
+  end
```

Comment:
berks 7 brings in chef as a gem which breaks the spiffy ruby version stuff, so you probably do need to do this.